### PR TITLE
[DRC-598] Update the recce server to instance

### PR DESCRIPTION
**Pull request by popcornylu**
Review and change some wording from `recce server` and `recce instance`. 

- If the context is describing a recce server instance in the cloud, we will use "recce instance." 
- If it's about executing the recce server command, we will use "recce server."

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -83,3 +83,4 @@ Try to keep animated gifs no bigger than around 1MB. Only use multiple large gif
 
 
 
+sasfsdfsdf
```
Comment (ijac13) [inline readme.md:86]: what is this?

File: docs/docs/features/preset-checks.md [hunk 2]
```diff
@@ -41,7 +41,7 @@ To configure the preset checks, add the settings to the [recce config file](../r
 
 ### Recce Server
 
-1. When a new recce server is launched, all preset checks are automatically set up, but these checks are not executed at this time.
+1. When a new recce server instance is launched, all preset checks are automatically set up, but these checks are not executed at this time.
```
Comment (ijac13) [inline docs/docs/features/preset-checks.md:44]: what is "recce server instance"?

File: docs/docs/architecture/overview.md [hunk 3]
```diff
@@ -52,7 +52,7 @@ A run documents a single execution of a Task, capturing details such as the type
     --data-raw '{}'
     ```
 
-In the recce server, each run is recorded, so in theory, it is possible to access the run history. However, the current UI does not offer this feature. You can see the complete run history from the exported recce state.
+In the recce instance, each run is recorded, so in theory, it is possible to access the run history. However, the current UI does not offer this feature. You can see the complete run history from the exported recce state.
```
Comment (ijac13) [inline docs/docs/architecture/overview.md:55]: when do we use Recce or recce?

File: readme.md (hunk 1 above)
Comment (popcornylu) [reply readme.md]: Fixed

File: docs/docs/architecture/overview.md (hunk 3 above)
Comment (popcornylu) [reply docs/docs/architecture/overview.md]: Change all the recce instance to Recce instance

File: docs/docs/features/preset-checks.md (hunk 2 above)
Comment (popcornylu) [reply docs/docs/features/preset-checks.md]: Change all the "recce server instance" to "Recce instance"

**Pull request merged**